docs/topics/Gateway.md

```diff
@@ -283,7 +283,7 @@ Any [events not defined in an intent](#DOCS_TOPICS_GATEWAY/commands-and-events-g
 
 If you specify an `intent` value in your `IDENTIFY` payload that is *invalid*, the socket will close with a [`4013` close code](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/gateway-gateway-close-event-codes). An invalid intent is one that is not meaningful and not documented above.
 
-If you specify an `intent` value in your `IDENTIFY` payload that is *disallowed*, the socket will close with a [`4014` close code](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/gateway-gateway-close-event-codes). A disallowed intent is one which you have not enabled for your bot or one that your bot is not whitelisted to use.
+If you specify an `intent` value in your `IDENTIFY` payload that is *disallowed*, the socket will close with a [`4014` close code](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/gateway-gateway-close-event-codes). A disallowed intent is one which you have not enabled for your bot or one that your bot is not allowed to use.
```

Comment:
This is now redundant. "A disallowed intent is one which you have not enabled for your bot or one that your bot is not allowed to use." -> "You are disallowed to use a disallowed intent." It should explain why you are disallowed.

Reply:
maybe rewrite the entire paragraph to

```
If you specify an `intent` value in your `IDENTIFY` payload that is *disallowed*, the socket will close with a [`4014` close code](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/gateway-gateway-close-event-codes). A disallowed intent is a privileged intent that has not been activated for this bot. Bots in under 100 guilds can enable these intents in the bot tab of the developer dashboard, verified bots have to request discord to activate these for the bot (either at the time of verification or by contacting support post verification).
```

Reply:
Sounds good.